templates/taxi/manufacturer_list.html

```diff
@@ -2,16 +2,18 @@
 
 {% block content %}
     <h1>Manufacturer List
-    </h1>
+    <a href="{% url 'taxi:manufacturer-create' %}">+</a>
+  </h1>
```

Comment:
Make sure to add a blank line at the end of the file to comply with the code style checklist.